modelling/src/main/java/org/axonframework/modelling/saga/repository/jdbc/SagaSchema.java

```diff
@@ -26,37 +26,53 @@
  */
 public class SagaSchema {
 
-    private static final String DEFAULT_SAGA_ENTRY_TABLE = "SagaEntry";
-    private static final String DEFAULT_ASSOC_VALUE_ENTRY_TABLE = "AssociationValueEntry";
-
     private final String sagaEntryTable;
+    private final String serializedSagaColumn;
     private final String associationValueEntryTable;
+    private final String associationKeyColumn;
+    private final String associationValueColumn;
+    private final String sagaIdColumn;
+    private final String sagaTypeColumn;
 
     /**
      * Initialize SagaSchema with default values.
      */
     public SagaSchema() {
-        this(DEFAULT_SAGA_ENTRY_TABLE, DEFAULT_ASSOC_VALUE_ENTRY_TABLE);
+        this(builder());
     }
 
     /**
      * Initialize SagaSchema with custom locations for event entry tables.
      *
-     * @param sagaEntryTable              The name of the entry table
-     * @param associationValueEntryTable  The name of the association value table
+     * @param sagaEntryTable             The name of the entry table
+     * @param associationValueEntryTable The name of the association value table
+     * @deprecated use {@link Builder} instead
      */
+    @Deprecated
     public SagaSchema(String sagaEntryTable, String associationValueEntryTable) {
-        this.sagaEntryTable = sagaEntryTable;
-        this.associationValueEntryTable = associationValueEntryTable;
+        this(builder()
+                .sagaEntryTable(sagaEntryTable)
+                .associationValueEntryTable(associationValueEntryTable)
+        );
+    }
+
+    private SagaSchema(Builder builder) {
```

Comment:
Nit: if we make this constructor `protected`, there's a change to extend from the `SagaSchema`.
Granted, I assume the chances are slim this will be used.
Nonetheless, virtually all infra components in Axon Framework using the builder paradigm have a `protected` constructor.
```suggestion
    protected SagaSchema(Builder builder) {
```

Reply:
I will make it protected.
I already have made change in my PR -> I did not push yet